kyo-sttp/native/src/main/scala/kyo/PlatformBackend.scala

```diff
@@ -0,0 +1,16 @@
+package kyo
+
+import kyo.Requests.Backend
+import sttp.client3.*
+
+object PlatformBackend:
```

Comment:
`kyo-sttp` doesn't have tests for JS and I'm also not introducing tests for Native. The challenge is how to provide a server for the tests. I'll open a ticket for it.